nncf/torch/quantization/layers.py

```diff
@@ -428,14 +429,31 @@ def _prepare_fq_export_quantization(self, x: torch.Tensor):
             levels = level_high - level_low + 1
         return x, levels, input_low, input_high
 
-    def _prepare_qdq_export_quantization(self, x: torch.Tensor):
+    def _prepare_qdq_export_quantization(self, x: torch.Tensor) -> Tuple[torch.Tensor, torch.Tensor, torch.Tensor, int]:
         x, level_high, level_low, input_low, input_high = self._prepare_export_quantization(x)
         with no_jit_trace():
             y_scale, y_zero_point = get_scale_zp_from_input_low_input_high(level_low,
                                                                            level_high,
                                                                            input_low,
                                                                            input_high)
-        return x, y_scale, y_zero_point
+            possible_axes = self._possible_per_channel_dimensions()
```

Comment:
Suggest calling the method like `get_channel_axis()` and encapsulate all the checks there.